packages/core/src/storages/utils.ts

```diff
@@ -22,3 +24,30 @@ export async function purgeDefaultStorages(config = Configuration.getGlobalConfi
         await client.purge?.();
     }
 }
+
+interface UseStateOptions {
```

Comment:
lets export this, and use just `config` instead of `storeManagerOptions`, this would result in unnecessary nesting